build.solr.xml

```diff
@@ -101,24 +101,41 @@
             </then>
         </if>
         <if>
-            <available file="${solr.core.dir}" type="dir" property="solr.core.dir.available" />
+            <available file="${solr.core.published.dir}" type="dir" property="solr.core.published.dir.available" />
             <then>
-                <echo message="A Solr core has already been configured at ${solr.core.dir}. Skipping." />
+                <echo message="A Solr core has already been configured at ${solr.core.published.dir}. Skipping." />
             </then>
             <else>
                 <!-- Create the Solr core. -->
                 <exec dir="${solr.dir}"
-                      command="./bin/solr create_core -c ${solr.core}"
+                      command="./bin/solr create_core -c ${solr.core.published}"
                       checkreturn="true"
                       passthru="true"/>
                 <!-- Copy the configuration. -->
-                <copy todir="${solr.core.dir}/conf" overwrite="true">
+                <copy todir="${solr.core.published.dir}/conf" overwrite="true">
                     <fileset dir="${solr.config.dir}" />
                 </copy>
-                <!-- Restart Solr so our configuration changes are picked up. -->
-                <phingcall target="restart-apache-solr" />
             </else>
         </if>
+        <if>
+            <available file="${solr.core.unpublished.dir}" type="dir" property="solr.core.unpublished.dir.available" />
```

Comment:
I left it like this only because I would have to put the definitions in separate variables in order to group them and then if in the future we 'd have to add extra data in only one of the cores, we would have to revert simple work that doesn't hurt us having a bit duplicated. It is just 2 commands working in different directories etc.